### ISSUE TITLE
stamp font cut off

### ISSUE THREAD

**Issue by tcurdt** [pdfcpu/pdfcpu#173]
I've stamped a PDF like this:

```
f:Helvetica, points:12, s:1 abs, pos:bl, off: 100 20, c:0 0 0, rot:0, op:1, m:0
```
but the parts of the text is cut off
![Screen Shot 2020-03-13 at 15 17 51](https://user-images.githubusercontent.com/13697/76629503-a5aa4a80-653e-11ea-812f-504965ee8b27.png)

The only way I found around this is to add a newline at the end and then adjust the position.
Did I do something wrong or is this a bug?

**Comment by hhrutter** [comment]
Possibly a bug.
I will revisit the vertical text layouting.

**Comment by hhrutter** [comment]
Fixed with [v0.3.3](https://github.com/pdfcpu/pdfcpu/releases)
